habitat_sim/simulator.py

```diff
@@ -297,14 +298,23 @@ def get_observation(self):
             if self._spec.gpu2gpu_transfer:
                 return self._gl_tensor.tensor().flip(0).squeeze(-1).clone()
             else:
+                size = mn.Vector2i(self._spec.resolution[1], self._spec.resolution[0])
```

Comment:
```suggestion
                size = self.__sensor_object.framebuffer_size
```

Reply:
Ah! I *knew* it was there somewhere :)

Reply:
Yeah, it should probably be on the rendering target also :)

Reply:
Ultimately the rendering target could expose the whole Framebuffer class, I think. Or (crazy thinking) be implemented completely in Python.